openstates/settings.py

```diff
@@ -59,3 +63,6 @@
         "boto": {"handlers": ["default"], "level": "WARN", "propagate": False},
     },
 }
+
+# Realtime Upload
+DATA_CLASSES = ["bill", "event", "vote_event", "jurisdiction", "organization"]
```

Comment:
This is a little vague as a setting-level variable, I'd suggest `DATA_CLASSES_TO_ARCHIVE` or `ARCHIVE_DATA_CLASSES` or something (corresponding to the `archive` flag ... or `REALTIME_DATA_CLASSES` if you decide to go wtih the `realtime` flag)